docs/src/jsx/CodeEditor.js

```diff
@@ -59,6 +60,9 @@ const StyledError = styled(LiveError)`
   padding: 1rem;
```

Comment:
Is prettier not formatting these? Seems the line above has inconsistent indentation. Maybe it only does so for me because I have a VS Code plugin...

Reply:
Hm the file looks fine to me? And I also don't see tabs mixed with spaces or so?

Reply:
It's actually line 58 that I was commenting about, it doesn't show up in the PR unless you click the button to expand.

![image](https://user-images.githubusercontent.com/14237/50319861-6a75b480-047e-11e9-96f6-f351a55b6d5d.png)

Reply:
Hah weird looks like a github rendering bug? There are only two spaces in front of both lines!